string label
float32 upper_x
float32 upper_y
float32 lower_x
float32 lower_y
float32 confidence
bool boolean